src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const FORMAT_SOURCE = sourceString.split('\n').join('');
+  const STYLE_ARRAY = FORMAT_SOURCE.split(';');
+
+  const STYLE_OBJECT = {};
+
+  const PAIRS_OF_STYLE = STYLE_ARRAY.map((item) => item.split(':'));
+
+  PAIRS_OF_STYLE.forEach((item) => {
+    if (item.length < 2) {
+      return 0;
+    }
+
+    const KEY = item[0].trim();
+    const VALUE = item[1].split(',').join(`,\n`).trim();
+
+    STYLE_OBJECT[KEY] = VALUE;
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Also, the `forEach` method is used correctly here, however, the return value `0` is not necessary. `forEach` does not use the return value of the callback function.